src/transformers/__init__.py

```diff
@@ -376,6 +378,8 @@
     _import_structure["models.mbart50"].append("MBart50Tokenizer")
     _import_structure["models.mt5"].append("MT5Tokenizer")
     _import_structure["models.pegasus"].append("PegasusTokenizer")
+    _import_structure["models.plbart"].append("PLBartTokenizer")
```

Comment:
Do we really need two tokenizers here?

Reply:
@patrickvonplaten There are two different types of models. One is trained with 7 languages, and one with 3 languages. `MultiTokenizer` is with 7 languages, so we need different language codes for that, hence I have added two tokenizers.

Reply:
@patil-suraj There is actually another difference. The `MultiTokenizer` is only used for fine-tuning purposes. Hence, it does not have `<mask>` token. Also, in the case of the regular tokenizer, there is a need for allowing no language token, but for `MultiTokenizer` there will always be a language code.
@wasiahmad Please correct me if I am wrong here.

Reply:
@gchhablani You are right. As shown [here](https://github.com/wasiahmad/PLBART/blob/main/multilingual/plbart/convert.py), for multilingual fine-tuning, we removed the `<mask>` token and added 4 new language tokens. 
I remember, in HF tokenizer, we can add special tokens using the `additional_special_tokens`, so for the multilingual fine-tuned checkpoint, we may use this parameter to add the language tokens?
Also, is it possible to exclude the `<mask>` token from the tokenizer by setting `mask_token = None`? In such a case, we may combine the two tokenizers. Please correct me if I am wrong, I do not know what happens with the `<mask>` token during downstream task fine-tuning.

Reply:
@patil-suraj In this case, is it possible to accomodate both of them together? Wdyt?

Reply:
Thanks @wasiahmad!
And yes, we can still accommodate this within a single tokenizer. As Wasi said, we can use the `additional_special_tokens` for the language tokens, for ex: that's what `M2M100Tokenizer` does. 

For this, we could add an argument like `use_language_codes` and if it's `True` add language codes else don't add.

Same for the `<mask>` token. Setting `mask_token = None` should work if the mask token is not required.